commonskmm/build.gradle.kts

```diff
@@ -54,6 +54,11 @@ kotlin {
 
             // Atomicfu
             implementation(libs.atomicfu)
+
+            // Sentry
+            api(libs.analytics.sentry)
```

Comment:
so if you have implemented Sentry on Android, why does commonMain has Sentry dependencies?